rpc/getSupply.go

```diff
@@ -21,20 +21,35 @@ import (
 )
 
 // GetSupply returns information about the current supply.
-func (cl *Client) GetSupply(
+func (cl *Client) GetSupply(ctx context.Context) (out *GetSupplyResult, err error) {
```

Comment:
I'd like to keep the existing `GetSupply` signature as-is (i.e. with the commitment param).

Reply:
```suggestion
func (cl *Client) GetSupply(ctx context.Context, commitment rpc.CommitmentType) (out *GetSupplyResult, err error) {
```